assets/js/components/OverlayCard/MainBody.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * OverlayCard MainBody component.
+ *
+ * Site Kit by Google, Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { Fragment } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { BREAKPOINT_SMALL, useBreakpoint } from '../../hooks/useBreakpoint';
+import Title from './Title';
+import Description from './Description';
+import DismissButton from './DismissButton';
+import CTAButton from './CTAButton';
+import { buttonProps } from '.';
+
+export default function MainBody( {
```

Comment:
Let's rename this component to be `Card` or just `Body` because `MainBody` sounds like there are more than just one body.